test/__snapshots__/index.test.js.snap

```diff
@@ -719,12 +719,12 @@ exports[`Validation should fail validation for if/then/else #4 1`] = `
 
 exports[`Validation should fail validation for if/then/else #5 1`] = `
 "Invalid configuration object. Object has been initialised using a configuration object that does not match the API schema.
- - configuration.ifThenElseKeyword2 should be <= 1000."
+ - configuration.ifThenElseKeyword2 should be <= 1000 (should be >= 1 and <= 1000)."
```

Comment:
Loos very misleading, i can't understand what we want :smile:

Reply:
You mean this is unnecessary? The problem is when error fires on `maximum` `minimum` `exclusive*` than we don't print parent schema. E.g. we have `5 <= x < 6` we know that x possible only 5, but we print `should be < 6` since error fired on `exclusiveMaximum` and we don't use info about parent schema

Reply:
this code https://github.com/webpack/schema-utils/blob/master/src/ValidationError.js#L746

Reply:
maybe we don't need to print this twice as here. I agree with this opinion

Reply:
yes twice looks very misleading

Reply:
fixed